fabric-connector/src/main/java/com/luxoft/fabric/config/AbstractConfigAdapter.java

```diff
@@ -0,0 +1,36 @@
+package com.luxoft.fabric.config;
+
+
+import com.luxoft.fabric.EventTracker;
+import org.hyperledger.fabric.sdk.User;
+
+public abstract class AbstractConfigAdapter implements ConfigAdapter {
+
+    private String defaultChannelName;
+    private User user;
+    private boolean initChannels;
```

Comment:
It seems to me, that `initChannels` is never used. The better approach would be drop `initChannels` completely, and eventually generalize EventTracker interface.

Reply:
Done